doc/classes/AudioServer.xml

```diff
@@ -344,5 +359,18 @@
 		<constant name="SPEAKER_SURROUND_71" value="3" enum="SpeakerMode">
 			A 7.1 channel surround setup was detected.
 		</constant>
+		<constant name="PLAYBACK_TYPE_DEFAULT" value="0" enum="PlaybackType" experimental="">
+			The playback will be considered of the type declared at [member ProjectSettings.audio/general/default_playback_type].
+		</constant>
+		<constant name="PLAYBACK_TYPE_STREAM" value="1" enum="PlaybackType" experimental="">
+			Force the playback to be considered as a stream.
+		</constant>
+		<constant name="PLAYBACK_TYPE_SAMPLE" value="2" enum="PlaybackType" experimental="">
+			Force the playback to be considered as a sample.
+			Be careful, as not every platform supports samples.
```

Comment:
```suggestion
			Force the playback to be considered as a sample.
			[b]Note:[/b] Only supported on the web platform. Other platforms will fall back to [constant PLAYBACK_TYPE_STREAM].
```

(Is the fallback behavior correct?)

Reply:
I could maybe introduce a fallback behavior, but currently, the sound just doesn't play (it should display an error, though).

Reply:
Crackling* by the way.